js/stateless.js/tests/e2e/compress.test.ts

```diff
@@ -0,0 +1,119 @@
+import { describe, it, assert, beforeAll } from 'vitest';
+import {
+    CompressedAccount,
+    CompressedAccountWithMerkleContext,
+    MerkleContext,
+    bn,
+} from '../../src/state';
+import { sendAndConfirmTx, buildAndSignTx } from '../../src/utils';
+
+import { Keypair, Signer } from '@solana/web3.js';
+import { defaultTestStateTreeAccounts } from '../../src/constants';
+import { getTestRpc, newAccountWithLamports } from '../../src/test-utils';
+import { LightSystemProgram, Rpc } from '../../src';
+
+describe('compress', () => {
+    const { merkleTree, nullifierQueue } = defaultTestStateTreeAccounts();
+    let rpc: Rpc;
+    let payer: Signer;
+    let bob: Signer;
+
+    beforeAll(async () => {
+        rpc = await getTestRpc();
+        payer = await newAccountWithLamports(rpc);
+        bob = Keypair.generate();
+    });
+    // Note:
+    // We don't compress SOL yet, therefore cannot spend utxos with value yet.
+    // TODO: add one run with with inputUtxo where lamports: 0
+    it('should compress lamports and then decompress', async () => {
+        const ix = await LightSystemProgram.initCompressedSolPda(
+            payer.publicKey,
+        );
+        const { blockhash: initBlockhash } = await rpc.getLatestBlockhash();
+        const signedInitTx = buildAndSignTx([ix], payer, initBlockhash);
+        await sendAndConfirmTx(rpc, signedInitTx);
+
+        const ixs = await LightSystemProgram.compress({
+            payer: payer.publicKey,
+            address: payer.publicKey,
+            lamports: 20,
```

Comment:
refactoring the lamports amount to a variable would make the test easier to maintain